.github/workflows/publish-draft-releases.yml

```diff
@@ -69,7 +69,7 @@ jobs:
       -
         name: build runtime
         id: srtool-build
-        uses: chevdor/srtool-actions@v0.3.0
+        uses: chevdor/srtool-actions@v0.4.0
```

Comment:
Why are we using chevdor 0.4 when parity's fork has reached v1.80?
https://github.com/paritytech/srtool/releases

Can we just switch?

Reply:
It's just a github action tool, the real tool locates at line 74 `image: paritytech/srtool`